src/templates/web/base.html

```diff
@@ -0,0 +1,241 @@
+        <div class="container-fluid">
+
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="#">Inicio</a>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
+                            data-bs-toggle="dropdown" aria-expanded="false">
+                            Categorías
+                        </a>
+                        <ul class="dropdown-menu " aria-labelledby="navbarDropdown ">
+                            
+                            {% for i in categoria%}
+                            <li><a class="dropdown-item" href="#">{{i}}</a></li>
+                            {%endfor%}
+                           
+                            <li>
+                                <hr class="dropdown-divider">
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
+                            data-bs-toggle="dropdown" aria-expanded="false">
+                            Editoriales
+                        </a>
+                        <ul class="dropdown-menu " aria-labelledby="navbarDropdown ">
+                            {% for i in editorial%}
+                                <li><a class="dropdown-item" href="#">{{i}}</a></li>
+                            {%endfor%}
+
+                            <li>
+                                <hr class="dropdown-divider">
+                            </li>
+                            <li><a class="dropdown-item" href="#">Todas las editoriales</a></li>
+                        </ul>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Quienes somos</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contactenos</a>
+                    </li>
+                </ul>
+                <ul class="navbar-nav ms-auto">
+                    <button class="btn"><i class="fa-solid fa-user"></i></button>
+                </ul>
+
+            </div>
+        </div>
+    </nav>
+    {% endblock %}
+
+    <div class="container py-5">
+        {% block body %}
+            {% for i in categoria %}
```

Comment:
recuerda quitar este for y dejar el bloque vacío, esto lo hicimos sólo como prueba rápida